README.md

```diff
@@ -23,6 +23,14 @@ See important security warnings in
 
 **Zcash is unfinished and highly experimental.** Use at your own risk.
 
+Deprecation Policy
+------------------
+
+This release is considered deprecated 16 weeks after the release day. There
```

Comment:
perhaps add "approximately" before 16 weeks, as it's based on block height and might be a little different?